sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/TrainModelWithLabels.java

```diff
@@ -36,13 +38,18 @@ public static void main(String[] args) {
         // Train custom model
         String trainingFilesUrl = "{SAS_URL_of_your_container_in_blob_storage}";
         // The shared access signature (SAS) Url of your Azure Blob Storage container with your forms.
-        SyncPoller<FormRecognizerOperationResult, CustomFormModel> trainingPoller = client.beginTraining(trainingFilesUrl, true);
+        SyncPoller<FormRecognizerOperationResult, CustomFormModel> trainingPoller
+            = client.beginTraining(trainingFilesUrl,
+            true,
+            new TrainingOptions().setModelName("composed model name"),
```

Comment:
since we're just calling beginTraining here, maybe we should name the model "model trained with labels"?